account_export_csv/tests/test_account_export_csv.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```

Comment:
utf-8 line can be removed in python3. Just safe the file as utf8 encoding is enough.

Reply:
This PR is for Odoo 10.0 that still needs python2, so it is correct to keep this line. Will be removed when migrating the module to 11.0.

Reply:
My bad!